docs-src/concepts/what-is-the-temporal-cli.md

```diff
@@ -28,3 +26,187 @@ Workflow, show Workflow History, and Signal Workflow).
 - [temporal server](/cli/server)
 - [temporal task-queue](/cli/task-queue/)
 - [temporal workflow](/cli/workflow/)
+
+## Installation
```

Comment:
Please remove the installation instructions if you're linking to the reviewed guide.